src/divider.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
+import ThemeManager from './styles/theme-manager';
+import styleUtils from './utils/styles';
+
+const propTypes = {
+  /**
+   * CSS class that will be add to the divider's root element
+   */
+  className: React.PropTypes.string,
+
+  /**
+   * If true, the divider will be indented by 72px
+   */
+  inset: React.PropTypes.bool,
+
+  /**
+   * Override the inline-styles of the list divider's root element.
+   */
+  style: React.PropTypes.object,
+};
+
+const contextTypes = {
+  muiTheme: React.PropTypes.object,
+};
+
+const childContextTypes = {
+  muiTheme: React.PropTypes.object,
+};
+
+const defaultProps = {
+  inset: false,
+};
+
+const Divider = ({inset, style, ...other}, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
```

Comment:
I think essentially this:
```js
 {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}
```
Does the same thing. I think if `muiTheme` is in context, it will use it. If it is undefined, it will default to `ThemeManager.getMuiTheme(DefaultRawTheme)`. 

I tested the code without the `=  ThemeManager.getMuiTheme(DefaultRawTheme)` and it still works.

I'll be honest, I'd love to tell you that I know for sure this is the same exact behavior as what we were doing before, but I'm not sure. But based on the docs I'm reading and what I'm seeing with the code, it looks like it's working.